.gcloudignore

```diff
@@ -11,19 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
We don't need these changes anymore, since #3404 is working 👍🏻

Reply:
I have reverted the change in .gcloudignore.